conbench/entities/history.py

```diff
@@ -168,6 +170,7 @@ def get_history_for_cchr(
             BenchmarkResult.data,
             BenchmarkResult.times,
             BenchmarkResult.change_annotations,
+            BenchmarkResult.timestamp.label("result_timestamp"),
```

Comment:
what does `label("result_timestamp")` do?

Reply:
Pretty sure it gets translated to `... AS result_timestamp`, which we need here because the commit timestamp is already called `timestamp`.